SignalServiceKit/src/Contacts/PhoneNumber.m

```diff
@@ -372,25 +372,23 @@ - (NSNumber *)getCountryCode {
     return self.phoneNumber.countryCode;
 }
 
-- (BOOL)isValid {
-    return [[PhoneNumberUtil sharedThreadLocal].nbPhoneNumberUtil isValidNumber:self.phoneNumber];
-}
-
-- (NSString *)localizedDescriptionForUser {
```

Comment:
This diff is confusing because I removed some dead code localizedDescriptionForUser and resolvesInternationallyTo